src/System.Management.Automation/engine/ExperimentalFeature/ExperimentalFeature.cs

```diff
@@ -108,6 +108,9 @@ static ExperimentalFeature()
                 new ExperimentalFeature(
```

Comment:
@PaulHigin, your last commit had 1 failures in `PowerShell-CI-macos`
Get-Variable.Should be able to use wildcard characters in the Name field
```powershell
Expected 4, but got 'Output:1'.
at <ScriptBlock>, /Users/vsts/agent/2.155.1/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Get-Variable.Tests.ps1: line 80
80: 		(Get-Variable -Name var*).Value[0] | Should -Be 4
```